python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(cls.gangadir, ignore_errors=True)
```

Comment:
I'm not sure this will still work as in `setUp` you're setting an *object* attribute `gangadir` but here you are reading the *class* attribute `gangadir` which are different. Put a `print(cls.gangadir)` in `tearDown` and run `nosetests` with `--nocapture` to see that it's always set to `None`.